### ISSUE TITLE
[Feature] Any plans for adding dashboard authentication?

### ISSUE THREAD

**Issue by felipecsl** [jobrunr/jobrunr#66]
I don't want my dashboard to be publicly visible to the world

**Describe the solution you'd like**
Simple basic auth (username/password) would suffice to protect it.

**Describe alternatives you've considered**
Could do IP whitelisting I guess or some other workaround.

**Comment by rdehuyss** [comment]
Hi Felipe,

I was planning to add authentication support in the Pro version. See [https://www.jobrunr.io/en/documentation/pro/#upcoming-features](https://www.jobrunr.io/en/documentation/pro/#upcoming-features). 

This is always a difficult choice to make - put a feature in the free version or the pro version - but I feel like if I keep adding extra features to the free version, there is no incentive to buy the Pro version.

Is this acceptable for you?

Another easy workaround is to put an nginx/any other proxy... in front of JobRunr.

Cheers!

**Comment by felipecsl** [comment]
Fair enough, I think it makes sense to add it to the pro version. It's ultimately your call, though 😃 
Using nginx in front of it is actually a good idea. I've been doing IP whitelisting but since I have a dynamic IP, that's inconvenient. Nginx works great, thanks!

**Comment by dtrouillet** [comment]
Hello,

This feature is for me a must have. I really like your work, Jobrunr is a beautiful tool.
Unfortunately, I am developing an open source project (informatique-cdc/ebad) which cannot afford to purchase the pro version. So for me two solutions: fork your repo or change the solution. I'll see how to do it. In any case, the dashboard is a great idea.

Regards

**Comment by longkerdandy** [comment]
Sorry to ask, but where is the documentation about Authorization function

**Comment by esezeolivera** [comment]
Hi @rdehuyss,

I am just starting to use your solution and it's great!!! Congratulations for the the great work put in here.

Just wanted to double check whether the Basic Authentication feature is only available in the pro version as you mentioned in your comment above? Just wondering.

Thanks.

**Comment by rdehuyss** [comment]
It was contributed so it's available in the open-source version of JobRunr. See the link above in this issue ;-) :
> Add support to secure dashboard.

Feel free to star the project if you think it is great. Thanks!
